### DISCUSSION TITLE
PHP Fatal error: Uncaught Error: Class "Symfony\Component\Console\Output\AnsiColorMode" not found in phar:/vendor/symfony/console/Terminal.php:18

### DISCUSSION THREAD

**Discussion by glo45343** [symfony/symfony#48590] [Q&A]
### Symfony version(s) affected

6.2.1

### Description

PHP Fatal error: Uncaught Error: Class "Symfony\Component\Console\Output\AnsiColorMode" not found in phar:/vendor/symfony/console/Terminal.php:18

### How to reproduce

compile with https://github.com/box-project/box

Now run some commands with phar file

**Comment by glo45343** [comment]
composer version 2.2 and php version 8.1

**Comment by derrabus** [comment]
Moving to support because "Compile with box" is not really a reproducer.
